charts/templates/configmap.yaml

```diff
@@ -6,7 +6,12 @@ metadata:
   labels:
     {{- include "marklogic.labels" . | nindent 4 }}
 data:
+{{- if ne .Values.bootstrapClusterName "" }}
+  MARKLOGIC_BOOTSTRAP_HOST: {{ .Values.bootstrapClusterName }}
+  MARKLOGIC_JOIN_CLUSTER: "true"
+{{- else }}
   MARKLOGIC_BOOTSTRAP_HOST: {{ include "marklogic.fullname" . }}-0
+  MARKLOGIC_JOIN_CLUSTER: "false"
```

Comment:
If we provide "false" then if we set the replicaCount=2 it won't join the cluster. We might need to provide one more environment variable like MARKLOGIC_IS_BOOTSTRAP_CLUSTER, and check in the docker script whether we should join the cluster. Basically, we should not join when it's the first node of bootstrap cluster. For all other cases, we should join the cluster.

Reply:
resolved the scaling issue by updating the MARKLOGIC_BOOTSTRAP_HOST mapping.